webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const path = require('path');
+const TerserPlugin = require('terser-webpack-plugin');
+
+module.exports = {
+  target: 'web',
+
+  entry: {
+    'code-worker': './src/CodeViewer/worker/index.ts',
+  },
+
+  output: {
+    path: path.join(__dirname, 'dist/workers'),
+    libraryTarget: 'commonjs-module',
+  },
+
+  resolve: {
+    extensions: ['.tsx', '.ts', '.js', '.mjs'],
+  },
+
+  optimization: {
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          ecma: 2020,
+          module: true,
+          safari10: true,
```

Comment:
HahaHAHAHA I wondered why you brought in TerserPlugin lol.

Reply:
FYI - we don't need to support safari 10 (if that is actually what this does).